### PR TITLE
Add Symfony template

### PR DESCRIPTION
**Pull request by sci3ma**
Template to setup minimal environment to run unit and functional tests in Symfony using PHPUnit ([Symfony's PHPUnit Bridge](https://symfony.com/doc/current/components/phpunit_bridge.html))

Briefly the template does:
- run a environment with ubuntu latest version
- checkout your code in the environment
- add directory permissions
- create a empty sqlite database
- execute the tests via phpunit injecting a couple of environment variables in order to use the sqlite database created

### CODE REVIEW

File: ci/symfony.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+name: Symfony
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
```
Comment (andymckay) [inline ci/symfony.yml:7]:
```suggestion
    branches: [ $default-branch ]
  pull_request:
    branches: [ $default-branch ]
```

File: ci/symfony.yml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+jobs:
+  symfony-tests:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Install Dependencies
+      run: composer install -q --no-ansi --no-interaction --no-scripts --no-suggest --no-progress --prefer-dist
```
Comment (andymckay) [inline ci/symfony.yml:18]: When I try this I get 
```

Run composer install -q --no-ansi --no-interaction --no-scripts --no-suggest --no-progress --prefer-dist
  composer install -q --no-ansi --no-interaction --no-scripts --no-suggest --no-progress --prefer-dist
  shell: /bin/bash -e {0}
##[error]Process completed with exit code 1.
```

Suggesting that composer isn't present, where would a user get composer from?

**Comment by sci3ma** [comment]
@andymckay Sorry for late response, I'm sooo busy :( I assumed that if anyone uses Symfony Framework then has to have composer installed. Feel free to improve this PR with your chances as I don't know when I'll find some free time.

**Comment by andymckay** [comment]
> @andymckay Sorry for late response, I'm sooo busy :( I assumed that if anyone uses Symfony Framework then has to have composer installed. Feel free to improve this PR with your chances as I don't know when I'll find some free time.

Thanks for the reply @sci3ma and sorry for the slow reply on my side in the first place. At this point, we expect a starter workflow to work as well as possible out of the box and I imagine users would like help installing Symfony in their VM. If anyone else is willing to contribute to this PR, then we can move it forwards, otherwise I recommend we close it. We can always re-open it later if you get some time and want to work on it.

**Comment by TheGarious** [comment]
Hello, 

I can retake this PR. 

Maybe recreate a new pr ?

**Comment by sci3ma** [comment]
> Hello,
> 
> I can retake this PR.
> 
> Maybe recreate a new pr ?

Sure :) go ahead.

**Comment by TheGarious** [comment]
Good !

I create a pr a few days ago with this point.

Needed reviewal!

**Pull request closed without merging**